SDWebImage/Core/SDDiskCache.m

```diff
@@ -71,13 +71,16 @@ - (NSData *)dataForKey:(NSString *)key {
     }
     NSData *data = [NSData dataWithContentsOfFile:filePath options:self.config.diskCacheReadingOptions error:nil];
     if (data) {
+        [[NSURL fileURLWithPath:filePath] setResourceValue:[NSDate date] forKey:NSURLContentAccessDateKey error:nil];
```

Comment:
Is this time consuming ?

If so, need a extra `if (self.config.diskCacheExpireType == SDImageCacheConfigExpireTypeAccessDate`) guard

If not, it's OK

Reply:
It is quite fast, significantly faster than the data read in my (very quick) testing.

<img width="766" alt="Screenshot 2024-10-08 at 11 12 33 AM" src="https://github.com/user-attachments/assets/64737a41-1c1d-4cc1-b816-cf93c7fdf159">

Logged times are (1) time to read data, (2) time to write content access date, and (3) 2/1